smokescreen.go

```diff
@@ -217,17 +214,42 @@ func main() {
 		}
 	}
 
-	server := graceful.NewServer(&http.Server{
+	server := http.Server{
 		Handler: handler,
-	})
+	}
+	timeout := time.Second * time.Duration(60)
```

Comment:
I'd probably stick this in a constant at the top (and e.g., let `runServer` look at the constant directly):

`const exitTimeout = 60 * time.Second`

(you don't need to cast 60 to a `time.Duration`—it's an untyped constant)